web/testing/regression-tests/karma.conf.js

```diff
@@ -62,13 +62,18 @@ module.exports = function(config) {
     files: [
       // Test framework
       { pattern: path.join(keyman_basename, test_relative_path, 'test.html'), nocache: true },
+      
+      // KeymanWeb should come before the tests are loaded - test-runner.js in particular
+      // requires the `keyman` object.  At this point, we only include the main script itself.
+      { pattern: path.join(keyman_basename, test_relative_path, 'web', 'unminified', 'keymanweb.js'), watched: false, included: true, nocache: true },
```

Comment:
Maybe there's some difference between my local development machine and the build agents, but things broke rapidly when this include was missing.